src/equations/2d/compressible_euler.jl

```diff
@@ -383,6 +384,8 @@ function boundary_conditions_sedov_self_gravity(u_inner, orientation, direction,
 end
 
 # Apply source terms
+# TODO: Taal remove methods with the signature below?
+#       Or keep them as an option for possiby increased performance?
```

Comment:
```suggestion
```

Reply:
Currently, we need these methods for the old implementation to keep the "dual career" approach alive. Hence, I would like to keep this for now. The current implementation of Taal uses only the other (point-wise) method below.

Reply:
OK.